src/scripts/inclusion-bot.yaml

```diff
@@ -115,11 +130,19 @@ triggers:
     alternatives:
       - volatile
       - unpredictable
+    why: >
+      ":TRIGGER:" arose as a derogatory reference to people with motor control
+      impairments due to illness or substance use.  We want to avoid
+      stigmatizing mental illness and addiction.
 
   - matches:
       - derp
     alternatives:
       - oops
+    why: >
+      ":TRIGGER:" term arose as a derogatory reference to people with
```

Comment:
do you want the word "term" here?

Reply:
That's a good idea! And it reminds me that I never added more comments to the YAML doc to explain the new section.